src/Console/BaseCommand.php

```diff
@@ -108,6 +108,16 @@ public static function getDescription(): string
         return '';
     }
 
+    /**
+     * Determine if the command should be hidden from the help output.
+     *
+     * @return bool
+     */
+    public static function getHidden(): bool
```

Comment:
Should it be isHidden()?

It could also be formulated positive: isDiscoverable() default true.
Just an idea. Let's what others think.

Reply:
Do you want to fix this up?